File: MODUL-3/C/PRAK301-2310817220002-RAUDATULSHOLEHAH.c
#include <stdio.h>

int main()
{
    int x, y;
    printf("Input : \n");
    scanf("%d %d", &x, &y);

    if (x <= y)
    {
        printf("Urutan terkecil ke terbesar : %d, %d\n", x, y);
    }
    else
    {
        printf("Urutan terkecil ke terbesar : %d, %d\n", y, x);
    }

    return 0;
}

File: MODUL-3/PYTHON/PRAK301-2310817220002-RAUDATULSHOLEHAH.py
x = int(input("Input nilai pertama: "))
y = int(input("Input nilai kedua: "))
if x <= y:
    terkecil = x
    terbesar = y
else:
    terkecil = y
    terbesar = x

print(f"Urutan terkecil ke terbesar: {terkecil}, {terbesar}")